DigitalLearningSolutions.Web/ViewModels/Support/SupportViewModel.cs

```diff
@@ -12,11 +12,11 @@ string currentSystemBaseUrl
         {
             CurrentPage = currentPage;
             DlsSubApplication = dlsSubApplication;
-            SupportSideNavViewModel = new SupportSideNavViewModel(currentSystemBaseUrl);
+            CurrentSystemBaseUrl = currentSystemBaseUrl;
         }
 
         public SupportPage CurrentPage { get; set; }
         public DlsSubApplication DlsSubApplication { get; set; }
-        public SupportSideNavViewModel SupportSideNavViewModel { get; set; }
+        public string CurrentSystemBaseUrl { get; set; }

```

Comment:
Two thoughts on this class.

Firstly, this is currently both
* The base class for all page view models in the support area.
* The view model for the support page.

It may be worth dissociating those two in case someone wants to add to the Support page VM without changing the rest of the support area VMs.
So just make this an abstract `BaseSupportViewModel` and add an inheriting `SupportViewModel` for the support page.

Secondly, I think it's fine for the views to get `currentSystemBaseUrl` from `ConfigHelper` directly rather than having it passed in. Do you see any issue with that?

Reply:
Regarding the `ConfigHelper` the same comment applies as the other regarding it only containing extension methods and would require injecting the `IConfiguration` to make use of it.
For the other point yeah that makes sense, I've added the `base` class. Although I've not done this for the `Faq` model as this already has a base class `BaseSearchablePageViewModel` due to the searching so it instead inherits the interface `ISupportViewModel`. Something is niggling about all this setup but not sure it warrants much more time a moe extensive refactor. As it stands it generally makes sense and is clear enough on what is going on.

Reply:
Yeah, think I think your assessment there is on point.